### PR TITLE
Added datatypes for Python.

### PR DESCRIPTION
**Pull request by dhruvsomani**

### CODE REVIEW

File: data/DataType/bool_python.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+keyword: bool
+integer-min: '0'
+integer-max: '1'
+integer-signed: false
+literals:
+  - 'true'
```
Comment (Man-Jain) [inline data/DataType/bool_python.yaml]: This can be True and False. Rest looks good to me.

Comment (jayvdb) [reply data/DataType/bool_python.yaml]: quotes not needed.

must be `True` and `False`.

python doesnt have `true` or `false`

**Comment by KVGarg** [comment]
There are more data types in Python like Complex Numbers, Tuples, Dictionary and built-in functions like time, enum. So should they be added ?

**Comment by jayvdb** [comment]
No, they are not native data types

File: data/DataType/bool_python.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+keyword: bool
+integer-min: '0'
+integer-max: '1'
+integer-signed: false
+literals:
+  - True
```
Comment (jayvdb) [inline data/DataType/bool_python.yaml]: these need to be in quotes, otherwise they are interpreted as yaml booleans

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/43/commits/61622dc5c84c7cd4e73454a21051417b30b78920

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**